chemprop/featurizers/molecule.py

```diff
@@ -30,7 +30,7 @@ def __call__(self, mol: Chem.Mol) -> np.ndarray:
 
 class CountFeaturizerMixin:
     def __call__(self, mol: Chem.Mol) -> np.ndarray:
-        return self.F.GetCountFingerprintAsNumPy(mol)
+        return self.F.GetCountFingerprintAsNumPy(mol).astype(np.int32)
```

Comment:
Why do we need to specify `np.int32` here? What type does it return with this?

Reply:
I also added a test and found that rdkit.Chem.rdFingerprintGenerator.GetMorganGenerator().GetCountFingerprintAsNumPy(mol) returns type np.uint32 which for some reason torch.from_numpy complains TypeError: can't convert np.ndarray of type numpy.uint32. The only supported types are: float64, float32, float16, complex64, complex128, int64, int32, int16, int8, uint8, and bool. So I slapped on a .astype(np.int32) to fix that.